SU-HSDO/suhumsci
# feat(88,105,106,109): Sparkbox Sprint 4

# READY FOR REVIEW

## Summary
- [STN-88](https://sparkbox.atlassian.net/browse/STN-88) - #456
- [STN-105](https://sparkbox.atlassian.net/browse/STN-105) - #454
- [STN-106](https://sparkbox.atlassian.net/browse/STN-106) - #453
- [STN-109](https://github.com/SU-HSDO/suhumsci/pull/458) - #458

## Need Review By (Date)
asap (1/28)

## Urgency
high

## Steps to Test
- ✅ We've verified that this branch passes the code climate and circle CI tests against the develop branch. (except for the failing php unit test that Mike will add a commit for to this branch)

1. See other PRs if you'd like to test locally for those steps.
2. Verify code looking at the file changes here.
3. Go into humsci_basic folder directory and run `npm i` and `npm run test` to verify tests pass.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
